system/draft/TestAuditSuite.py

```diff
@@ -84,7 +102,7 @@ class TestAuditSuite:
         p1.start_service()
         for node in test_nodes[5:]:
             node.restart_service()
-        time.sleep(30)
+        await asyncio.sleep(30)
```

Comment:
Is sleep really needed here? If it is - can it be replaced by some `evenutal` check?